docs/userguide/icu4c/index.md

```diff
@@ -154,31 +154,28 @@ In the descriptions below, `<ICU>` is the full path name of the ICU4C directory
             <li><b>brkitr/</b> Data files for character, word, sentence, title
             casing and line boundary analysis.</li>
 
-            <li><b>coll/</b> Data for collation tailorings. The makefile
-            <b>colfiles.mk</b> contains the list of resource bundle files.</li>
+            <li><b>coll/</b> Data for collation tailorings.</li>
 
             <li><b>locales/</b> These .txt files contain ICU language and
             culture-specific localization data. Two special bundles are
             <b>root</b>, which is the fallback data and parent of other bundles,
-            and <b>index</b>, which contains a list of installed bundles. The
-            makefile <b>resfiles.mk</b> contains the list of resource bundle
-            files. Some of the locale data is split out into the type-specific
+            and <b>index</b>, which contains a list of installed bundles.
+            Some of the locale data is split out into the type-specific
             directories curr, lang, region, unit, and zone, described below.</li>
 
             <li><b>curr/</b> Locale data for currency symbols and names (including
-            plural forms), with its own makefile <b>resfiles.mk</b>.</li>
+            plural forms).</li>
 
             <li><b>lang/</b> Locale data for names of languages, scripts, and locale
-            key names and values, with its own makefile <b>resfiles.mk</b>.</li>
+            key names and values.</li>
 
             <li><b>region/</b> Locale data for names of regions, with its own
-            makefile <b>resfiles.mk</b>.</li>
+            makefile.</li>
```

Comment:
nit: Can we also remove "with its own makefile" here?

Reply:
Indeed 😅 I've just updated the PR accordingly, thanks!

Reply:
Thank you! We'll also need to squash the commits into a single commit that has the ticket number in the message, or else our PR checks won't pass.